apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -351,7 +354,7 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
                     if (distributionReceiptsIds.length !== uniqueDistributionReceiptsIds.size) {
                         throw new GQLError(ERRORS.BAD_AMOUNT_DISTRIBUTION_FOR_RECEIPTS, context)
                     }
-                    if (distributionReceiptsAmounts.filter(amount => amount.lte(MINIMUM_PAYMENT_AMOUNT)).length > 0) {
+                    if (distributionReceiptsAmounts.filter(amount => amount.lte(0)).length > 0) {
```

Comment:
why `lte` here, when in other modules you throw error on `lt` ?

Reply:
+ why 0 here?

Reply:
When user want to make a partial payment he needs to pass amounts for every payment he choose. Agreement with mobile APP